docs/content/figures.md

```diff
@@ -18,27 +81,86 @@ Here is my figure caption!
 ```{figure} ../images/C-3PO_droid.png
 ---
 height: 150px
-name: my-figure
+name: directive-fig
 ---
 Here is my figure caption!
 ```
 
 ```{note}
 You can also include figures that were generated from your code in notebooks.
-To do so, see {doc}`glue`.
+To do so, see [](content:code-outputs:glue).
+```
+
+## Markdown Figures
+
+Markdown figures combine [colon style admonitions](admonitions:colons) and [HTML image parsing](content-blocks-images), to produce a "Markdown friendly" syntax for figures,
+with equivalent behaviour to the `figure` directive above.
+
+To enable them, add in your `_config.yml`:
+
+```yaml
+parse:
+  myst_extended_syntax: true
 ```
 
+The figure block must contain **only** two components; an image, in either Markdown or HTML syntax, and a single paragraph for the caption.
+
+As with admonitions, the figure can have additional classes set on it, but the title is now taken as the reference target of the figure:
+
+```md
+:::{figure,myclass} markdown-fig
+<img src="../images/fun-fish.png" alt="fishy" class="bg-primary mb-1" width="200px">
+
+This is a caption in **Markdown**!
+:::
+```
+
+:::{figure,myclass} markdown-fig
+<img src="../images/fun-fish.png" alt="fishy" class="bg-primary mb-1" width="200px">
+
+This is a caption in **Markdown**!
+:::
+
+As we see here, the target we set can be referenced:
+
+```md
+[Go to the fish!](markdown-fig)
+```
+
+[Go to the fish!](markdown-fig)
+
 ## Referencing figures
 
-You can then refer to this figure using the `{ref}` role like:
-`` {ref}`my-figure` ``, which will replace the reference with the figure
-caption like so: {ref}`my-figure`.
+You can then refer to your figures using the `{ref}` role or Markdown style references like:
+
+```md
+- {ref}`directive-fig`
+- [](markdown-fig)
+```
+
+which will replace the reference with the figure caption like so:
+
+- {ref}`directive-fig`
+- [](markdown-fig)
+
 Another convenient way to create cross-references is with the `{numref}` role,
```

Comment:
I think we should have a header that has "numbered references" or something, so people find it more easily

Reply:
added in coming commit